src/server/db/localstorage.ts

```diff
@@ -0,0 +1,53 @@
+import { InMemory } from './inmemory';
+
+const globalAny: any = global;
+
+class WithLocalStorageMap<Tkey, Tvalue> extends Map {
+  key: string;
+  constructor(key: string) {
+    super();
+    const inCache = initFromCache(key);
+    inCache.forEach((pair: any[]) => {
+      this.set(pair[0], pair[1]);
+    });
+    this.key = getStorageKey(key);
+  }
+  sync(): void {
+    globalAny.localStorage.setItem(
```

Comment:
What's the purpose of `globalAny`? Is it just to avoid adding an interface for `localStorage`?

Reply:
Ah, thats because im typescript novice, and when i implemeted this outside the repo i had some error from ts about `locaStorage` usage most probably related to something in `tsconfig`. Will remove.